src/ser.rs

```diff
@@ -366,13 +381,8 @@ impl<'s> ser::Serializer for &'s Serializer {
         ))
     }
 
-    /// Serialises Rust maps into JS `Map`.
```

Comment:
Please update the doc comment instead of removing :)

Reply:
By "update" I meant it should say something like "or plain JS object, depending on configuration of `serialize_maps_as_objects`" (feel free to reword).